### PR TITLE
[fix][doc] Fixed wrong text in Full Transitive and Full Schema Compatibility docs

### PR DESCRIPTION
**Pull request by asafm**
### Modifications
Fix wrong text in Full and Full Transitive schema compatibility 

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `no-need-doc` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@Anonymitaet Please help review this PR. I noticed there are the format changes.

**Review by Technoboy- — APPROVED** [review]

**Review by Anonymitaet — COMMENTED** [review]
Hi @asafm  preview your changes locally and attach the preview screenshots in your PR description, steps: https://docs.google.com/document/d/1wszdtMRo6MhKbVaggPK7_bnKaC4TewuT--GWZZxJNGg/edit#

File: site2/docs/schema-evolution-compatibility.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ In some data formats, for example, Avro, you can define fields with default valu
 
 :::tip
 
-You can set schema compatibility check strategy at the topic, namespace or broker level. For how to set the strategy, see [here](schema-manage.md/#set-schema-compatibility-check-strategy).
+You can set schema compatibility check strategy at the topic, namespace or broker level. For how to set the strategy, see [here](schema-manage.md#set-schema-compatibility-check-strategy).
```
Comment (asafm) [inline site2/docs/schema-evolution-compatibility.md:158]: Preview
image

Leading to 
image

**Comment by asafm** [comment]
> Hi @asafm please preview your changes locally and attach the preview screenshots in your PR description, steps: https://docs.google.com/document/d/1wszdtMRo6MhKbVaggPK7_bnKaC4TewuT--GWZZxJNGg/edit#

@Anonymitaet, I've pasted screenshots for each change as comments in the PR next to the change

**Pull request merged**